### PR TITLE
Add onboarding steps to workspace

### PR DESCRIPTION
**Pull request by djbarnwal**
Adds #426 

Screenshot 2022-06-29 at 1 43 55 PM

### CODE REVIEW

**Comment by ericpgreen2** [comment]
Sometimes I don't see the onboarding messages show up. I've run these commands to test it:
```bash
npm install-and-build
npm run cli -- start --project NEWFOLDER
```
And the workspace is blank. Only when I add a source & model, then delete the model, do the onboarding steps appear.

File: src/routes/_surfaces/workspace/Onboarding.svelte [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  import ParquetIcon from "$lib/components/icons/Parquet.svelte";
+  import Code from "$lib/components/icons/Code.svelte";
+  // import Database from "$lib/components/icons/Database.svelte";
+  // import SurfaceView from "$lib/components/icons/SurfaceView.svelte";
+
+  const steps = [
+    {
+      heading: "import your data source",
+      description:
+        "Add to your sources by dragging a csv or parquet file to this window.",
```
Comment (ericpgreen2) [inline src/routes/_surfaces/workspace/Onboarding.svelte:11]: Can we also mention you can hit the "+" icon in the Sources list?

Comment (djbarnwal) [reply src/routes/_surfaces/workspace/Onboarding.svelte:11]: @magorlick sent me this copy. @magorlick should we also mention about the "+" icon?

**Comment by ericpgreen2** [comment]
Ah, yeah that's probably the "preferred" way to start the application (given the instructions in the ReadMe). But since it's also possible to start the application _without_ initializing a project, I think the onboarding page should also be displayed in that case. In fact, it's probably more important to show the onboarding steps in that case, because then we know the user isn't following the ReadMe, so we should put the instructions in the app itself.

**Comment by djbarnwal** [comment]
@ericpgreen2 Have updated the PR to work with `start` command as well

**Comment by magorlick** [comment]
@godzig prefers sentence case for headers FYI.

File: src/routes/_surfaces/workspace/Onboarding.svelte (hunk 1 above)
Comment (magorlick) [reply src/routes/_surfaces/workspace/Onboarding.svelte:11]: You can add "or clicking '+' under SOURCES" to the first bullet's text.

Comment (ericpgreen2) [reply src/routes/_surfaces/workspace/Onboarding.svelte]: Just got the green light from her, so I just made the edit

**Pull request merged**